### ISSUE TITLE
Running Tests

### ISSUE THREAD

**Issue by balasankarc** [rumpelsepp/rugments#2]
I tried running the tests using `bundle exec rake` (of course, after performing a bundle install), but got the following error. Can you help me fix it? The problem seems to be in loading the various Rouge Lexers.
`rugments-1.0.0.beta3 $ bundle exec rake`
ruby -I./lib -r ./spec/spec_helper -r./spec/cli_spec.rb -r./spec/formatters/html_spec.rb -r./spec/formatters/null_spec.rb -r./spec/formatters/terminal256_spec.rb -r./spec/lexer_spec.rb -r./spec/lexers/apache_spec.rb -r./spec/lexers/apple_script_spec.rb -r./spec/lexers/c_spec.rb -r./spec/lexers/clojure_spec.rb -r./spec/lexers/coffeescript_spec.rb -r./spec/lexers/common_lisp_spec.rb -r./spec/lexers/conf_spec.rb -r./spec/lexers/cpp_spec.rb -r./spec/lexers/csharp_spec.rb -r./spec/lexers/css_spec.rb -r./spec/lexers/dart_spec.rb -r./spec/lexers/diff_spec.rb -r./spec/lexers/elixir_spec.rb -r./spec/lexers/erb_spec.rb -r./spec/lexers/erlang_spec.rb -r./spec/lexers/factor_spec.rb -r./spec/lexers/gherkin_spec.rb -r./spec/lexers/go_spec.rb -r./spec/lexers/groovy_spec.rb -r./spec/lexers/haml_spec.rb -r./spec/lexers/handlebars_spec.rb -r./spec/lexers/haskell_spec.rb -r./spec/lexers/html_spec.rb -r./spec/lexers/ini_spec.rb -r./spec/lexers/io_spec.rb -r./spec/lexers/java_spec.rb -r./spec/lexers/javascript_spec.rb -r./spec/lexers/literate_coffeescript_spec.rb -r./spec/lexers/literate_haskell_spec.rb -r./spec/lexers/llvm_spec.rb -r./spec/lexers/lua_spec.rb -r./spec/lexers/make_spec.rb -r./spec/lexers/markdown_spec.rb -r./spec/lexers/matlab_spec.rb -r./spec/lexers/moonscript_spec.rb -r./spec/lexers/nginx_spec.rb -r./spec/lexers/nim_spec.rb -r./spec/lexers/objective_c_spec.rb -r./spec/lexers/ocaml_spec.rb -r./spec/lexers/perl_spec.rb -r./spec/lexers/plain_text_spec.rb -r./spec/lexers/prolog_spec.rb -r./spec/lexers/properties_spec.rb -r./spec/lexers/puppet_spec.rb -r./spec/lexers/python_spec.rb -r./spec/lexers/qml_spec.rb -r./spec/lexers/r_spec.rb -r./spec/lexers/racket_spec.rb -r./spec/lexers/ruby_spec.rb -r./spec/lexers/rust_spec.rb -r./spec/lexers/scala_spec.rb -r./spec/lexers/scheme_spec.rb -r./spec/lexers/sed_spec.rb -r./spec/lexers/shell_spec.rb -r./spec/lexers/slim_spec.rb -r./spec/lexers/smalltalk_spec.rb -r./spec/lexers/sml_spec.rb -r./spec/lexers/sql_spec.rb -r./spec/lexers/tcl_spec.rb -r./spec/lexers/tex_spec.rb -r./spec/lexers/toml_spec.rb -r./spec/lexers/viml_spec.rb -r./spec/lexers/xml_spec.rb -r./spec/lexers/yaml_spec.rb -r./spec/lexers_spec.rb -r./spec/plugins/redcarpet_spec.rb -r./spec/theme_spec.rb -r./spec/token_spec.rb -r./spec/visual_spec.rb -e Minitest::Unit.autorun
/home/balasankarc/packaging/rugments/ruby-rugments-1.0.0.beta3/spec/lexers/apache_spec.rb:1:in `<top (required)>': uninitialized constant Rouge::Lexers::Apache (NameError)
	from -e:1:in `require'
rake aborted!
Command failed with status (1): [ruby -I./lib -r ./spec/spec_helper -r./spe...]
/home/balasankarc/packaging/rugments/ruby-rugments-1.0.0.beta3/Rakefile:7:in `block in <top (required)>'
Tasks: TOP => default => spec
(See full trace by running task with --trace)

PS : I am packaging this gem for debian and need tests to be run successfully to accomplish that. So, please help me to get them run.

**Comment by rumpelsepp** [comment]
Tests are not ready yet. I'm sorry. Currently I am very busy as I am moving to Munich and getting a new job. So please give me some time and I will fix it. :)

**Comment by rumpelsepp** [comment]
As site note: on master I have started new tests; including CI. I have decided to create new tests because I want to change several internal things and it would be a mess to fix everything. The travis repo is here: https://travis-ci.org/rumpelsepp/rugments

**Comment by balasankarc** [comment]
Can you please comment here (or close this issue) when the tests are ready. Thanks. :)

**Comment by rumpelsepp** [comment]
Yeah, I have assigned it to milestone 1.0. So you will get noticed. :)
